MAIN/PresentationExporter.py

```diff
@@ -0,0 +1,25 @@
+# This class will handle saving the explanations for all slides in a JSON file.
+import json
+import os
+from dotenv import load_dotenv
+# Load environment variables from .env file
+load_dotenv()

```

Comment:
Expressions outside of a function are a bad habit. This code will be called every time the file is loaded and imported.   Read more here: https://www.geeksforgeeks.org/python-main-function/amp/